### PR TITLE
make light version agent for debugging

### PR DESCRIPTION
**Pull request by dnddnjs**
1. make minimal neural network for value and policy
2. just do reinforcement learning without tree search

now the loss function is minus. need to take a look at this
https://github.com/reinforcement-learning-kr/alphago-zero/issues/20

### CODE REVIEW

**Comment by kekmodel** [comment]
utils 의 코멘트에서 greedy하게 선택이 안된다는 것은 어떤 내용인가요?

**Comment by Kyushik** [comment]
어제 웅원님이랑 코드를 살펴봤는데 action을 선택할 때 tau에 따라 action이 2가지 선택을 하는 것 같던데 
1. pi max를 따라 선택하는 것 (Greedy)
2. random action을 선택 
이때 greedy한 action 선택이 아니라 확률적으로 선택하게 수정하신 것 같아요 ㅎㅎ

**Comment by Kyushik** [comment]
정대님의 질문 내용에 대해서는 이슈에 자세히 기록했습니다. 일단 이 부분까지는 merge 하도록 하겠습니다~ ㅎㅎ 
저도 연구실 컴퓨터로 해당 내용에 대해 돌려보고 결과 공유할게요 ㅎㅎ
